rpm/sdk-setup.spec

```diff
@@ -8,7 +8,7 @@ License:    GPL
 BuildArch:  noarch
 URL:        https://github.com/sailfishos/sdk-setup
 Source0:    %{name}-%{version}.tar.bz2
-BuildRequires:  systemd
+BuildRequires:  pkgconfig(systemd)
```

Comment:
IIUC systemd is build required here just for its RPM macros like `%systemd_requires`, `%systemd_post` etc. It makes no use of pkgconfig.

So this makes me think:

1. Are those macros possibly packaged separately nowadays? If not, then
2. Is this change really required for all packages, including sdk-setup, or just those that may come into way at bootstrap time? If yes, then
3. Isn't there an alternative way to let it choose between systemd and systemd-mini, more suitable for cases like this?